src/JobQueue/Message/MessageSerializationInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Icecave\Overpass\JobQueue\Message;
+
+use Icecave\Overpass\JobQueue\Exception\InvalidMessageException;
+
+interface MessageSerializationInterface
+{
+    /**
+     * Serialize a task request message.
+     *
+     * @param Request $request
+     *
+     * @return string
+     */
+    public function serializeRequest(Request $request);
```

Comment:
We talked about calling this either Job or Task, can't remember which we decided on.

What about Task? A task request will result in a task being performed in order to complete a job.